drafts/0007-RFC-BAG-OPERATORS.md

```diff
@@ -44,279 +44,288 @@ INTERSECT(A, B) = DISTINCT(INTERSECT_ALL(A, B))
 EXCEPT(A, B) = DISTINCT(EXCEPT_ALL(A, B))
 ```
 
-The following examples show how these operators can be used to construct new heterogenous bags. All examples will use permissive typing mode as conventional typing mode is equivalent to the SQL bag operators.
+In strict-typing mode, the set operators behave just like the set operators defined in the SQL specification as discussed in the next section. Outside of strict-typing mode, these operators are the pure mathematical multi-set union, intersect, and except which enables the creation and combination of heterogneous bags (TODO polish this sentence).
+
+### SQL Compatibility
+
+In the presence of schema, the column descriptors for a table are known. These column descriptors are an ordered list of metadata objects which describe a column's: type, name, nullability, and more (see Appendix A). The column descriptors play an important role for set operators. Two relations *T1* and *T2* are said to be *union-compatible* if they have the same number of columns (\*) and the data type for the i-th column of *T1* and the data type for the i-th column of *T2* are comparable. Comparability and the resultant data type follow the same rules as the comparability rules and data types of results of aggregations defined in SQL-99 section 9.3.  
+
+Each set operator produces a new relation *TR* which has its own column descriptors. The exact details of the these column descriptors is discussed in Appendix A, as well as SQL-99 section 7.12. The specification states that, for SQL conformance:
+
+**Column Naming Rule**
+- If the i-th columns of *T1* and *T2* share the name *C*, then the i-th column of *TR* will be named *C*.
+- Otherwise, the name of the i-th column of *TR* is implementation-dependent and must be unique amongst all other column names.
```

Comment:
What is the common behavior for the implementation dependence?  Is there one?  Worth noting in case it makes sense to suggest an implementation strategy here.